x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -387,10 +157,11 @@ export const importRulesRoute = (
           }
         });
         const importRules: ImportRulesResponseSchema = {
-          success: errorsResp.length === 0,
-          success_count: successes.length,
-          errors: errorsResp,
+          success: errorsResp.length === 0 && exceptionsSuccess,
```

Comment:
In follow up PR plan to return the exceptions summary to differentiate between rule success/error and exceptions success/error.